src/client/src/services/projects/test.js

```diff
@@ -0,0 +1,34 @@
+import TaskService from './index';
```

Comment:
Instead of TaskService, this would be ProjectService.

Reply:
Alright!